requirements/gateway-requirements.txt

```diff
@@ -3,7 +3,7 @@
 # dev/generate_requirements.py script to regenerate this file.
 
 pydantic<3,>=1.0
-fastapi<1
+fastapi<=0.109.2
```

Comment:
unrelated to keras, but the `deployments` CI job is also failing due to `fastapi` upgrading their version of `starlette` to 0.36.3: https://github.com/tiangolo/fastapi/pull/11086

`starlette>=0.36.0` now [throws when passed an invalid config file](https://github.com/encode/starlette/pull/2422), and `slowapi` dependency by [default passes](https://github.com/laurentS/slowapi/blob/master/slowapi/extension.py#L158) in `".env"` as a file if none is provided.